env/types.go

```diff
@@ -16,7 +16,181 @@ import (
+		"target width for the wrapped format",
+	},
+	{
+		"csv_fieldsep",
+		`field separator for CSV output (default ",")`,
+	},
+	{
+		"expanded",
+		"expanded output [on, off, auto]",
+	},
+	{
+		"fieldsep",
+		`field separator for unaligned output (default "|")`,
+	},
+	{
+		"fieldsep_zero",
+		"set field separator for unaligned output to a zero byte",
+	},
+	{
+		"footer",
+		"enable or disable display of the table footer [on, off]",
+	},
+	{
+		"format",
+		"set output format [unaligned, aligned, wrapped, vertical, html, asciidoc, csv, json, ...]",
+	},
+	{
+		"linestyle",
+		"set the border line drawing style [ascii, old-ascii, unicode]",
+	},
+	{
+		"null",
+		"set the string to be printed in place of a null value",
+	},
+	{
+		"numericlocale",
+		"enable display of a locale-specific character to separate groups of digits",
+	},
+	{
+		"pager_min_lines",
+		"minimum number of lines required in the output to use a pager, 0 to disable (default)",
+	},
+	{
+		"pager",
+		"control when an external pager is used [on, off, always]",
+	},
+	{
+		"recordsep",
+		"record (line) separator for unaligned output",
+	},
+	{
+		"recordsep_zero",
+		"set record separator for unaligned output to a zero byte",
+	},
+	{
+		"tableattr",
+		"specify attributes for table tag in html format, or proportional column widths for left-aligned data types in latex-longtable format",
+	},
+	{
+		"time",
+		`format used to display time/date column values (default "RFC3339Nano")`,
+	},
+	{
+		"title",
+		"set the table title for subsequently printed tables",
+	},
+	{
+		"tuples_only",
+		"if set, only actual table data is shown",
+	},
+	{
+		"unicode_border_linestyle",
+		"set the style of Unicode line drawing [single, double]",
+	},
+	{
+		"unicode_column_linestyle",
+		"set the style of Unicode line drawing [single, double]",
+	},
+	{
+		"unicode_header_linestyle",
+		"set the style of Unicode line drawing [single, double]",
+	},
+}
+
+var envVarNames = []varName{
+	{
+		text.CommandUpper() + "_EDITOR, EDITOR, VISUAL",
+		"editor used by the \\e, \\ef, and \\ev commands",
+	},
+	{
+		text.CommandUpper() + "_EDITOR_LINENUMBER_ARG",
+		"how to specify a line number when invoking the editor",
+	},
+	{
+		text.CommandUpper() + "_HISTORY",
+		"alternative location for the command history file",
+	},
+	{
+		text.CommandUpper() + "_PAGER, PAGER",
+		"name of external pager program",
+	},
+	{
+		text.CommandUpper() + "_SHOW_HOST_INFORMATION",
+		"display host information when connecting to a database",
+	},
+	{
+		text.CommandUpper() + "RC",
+		"alternative location for the user's .psqlrc file",
+	},
+	{
+		"SYNTAX_HL",
+		"enable syntax highlighting",
+	},
+	{
+		"SYNTAX_HL_FORMAT",
+		"chroma library formatter name",
+	},
+	{
+		"SYNTAX_HL_STYLE",
+		`chroma library style name (default "monokai")`,
+	},
+	{
+		"SYNTAX_HL_OVERRIDE_BG",
+		"enables overriding the background color of the chroma styles",
+	},
+	{
+		"SHELL",
+		"shell used by the \\! command",
+	},
+	{
+		"TMPDIR",
```

Comment:
I don't think this one is actually used anywhere?

Reply:
To clarify, I mean the `TMPDIR` variable.

Reply:
Removed.